### PR TITLE
Test

### PR DESCRIPTION
**Pull request by encosllo**
Proof of concept.

### CODE REVIEW

File: nouns.json [hunk 1]
```diff
@@ -1979,5 +2141,375 @@
...
+      "TR": { "atom": "", "gend": "" },
+      "ZH": { "atom": "", "gend": "" }
+    },
+    "adjs": {}
+  },
+  "120b03a0": {
+    "root": {
+      "CA": { "atom": "adherència", "gend": "femení" },
+      "DE": { "atom": "", "gend": "" },
+      "EN": { "atom": "adherence", "gend": "" },
```
Comment (thosgood) [inline nouns.json:2414]: is this "closure", as in "of a topological subset"? if so, I've never heard "adherence" used before, so maybe switch to "closure"? (and thus change the id to `0be302fe`)

File: nouns.json [hunk 2]
```diff
@@ -488,6 +488,168 @@
...
+        "PL": { "atom": "", "pstn": "" },
+        "PT": { "atom": "", "pstn": "" },
+        "RU": { "atom": "", "pstn": "" },
+        "TR": { "atom": "", "pstn": "" },
+        "ZH": { "atom": "", "pstn": "" }
+      },
+      "1ca304ab": {
+        "CA": { "atom": "simètrica", "pstn": "after" },
+        "DE": { "atom": "", "pstn": "" },
+        "EN": { "atom": "symmetrical", "pstn": "before" },
```
Comment (thosgood) [inline nouns.json]: it seems that "symmetric" is much more common that "symmetrical" (though I had to do a google search to be sure...), so could you change this one too please? (the new id would be `13b903de`)

**Review by thosgood — CHANGES_REQUESTED** [review]
two small changes to be made, if you agree with them :-)
(very exciting that you're starting to add stuff that I haven't added!)

File: nouns.json (hunk 1 above)
Comment (encosllo) [reply nouns.json:2414]: As far as I know, they are synonyms, and I'm much more used to adherence. I've read somewhere that you wanted to add this feature, so synonyms can point to each other. So, if you agree, I add closure as well and we can keep them both.

File: nouns.json (hunk 2 above)
Comment (encosllo) [reply nouns.json]: No problem with this one. Done.

File: nouns.json (hunk 1 above)
Comment (thosgood) [reply nouns.json:2414]: I need to think a bit more about how to deal with synonyms, it's true...

**Comment by encosllo** [comment]
Thanks!

**Pull request merged**